color-ui.ftd

```diff
@@ -1,14 +1,38 @@
 -- import: fpm
 
 
-
 --- header:
+color: $fpm.color
 
 --- primary-body:
+color: $fpm.color
 
 --- buttons-first-row:
+color: $fpm.color
 
 --- buttons-second-row:
+color: $fpm.color
+
+--- body-with-hover:
+color: $fpm.color
+
+--- toasts:
+color: $fpm.color
+
+--- normal-buttons:
+color: $fpm.color
+
+--- hover-buttons:
+color: $fpm.color
+
+--- pressed-buttons:
+color: $fpm.color
+
+--- focused-buttons:
+color: $fpm.color
+
+--- shadow-box:
+color: $fpm.color
```

Comment:
Remove these. What is the purpose it is serving? Also there can't be any subsections without section